pkg/common-controller/snapshot_controller_base.go

```diff
@@ -322,11 +330,11 @@ func (ctrl *csiSnapshotCommonController) snapshotWorker() {
 	}
 	defer ctrl.snapshotQueue.Done(keyObj)
 
-	if err := ctrl.syncSnapshotByKey(keyObj.(string)); err != nil {
+	if err := ctrl.syncSnapshotByKey(keyObj); err != nil {
```

Comment:
It's no longer `keyObj`, but directly `key`. Please update it in all workers.

Reply:
@jsafrane done, PTAL